src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -642,28 +646,25 @@ public static bool Equals(object first, object second, bool ignoreCase, IFormatP
 
             if (first == null)
             {
-                return IsNullLike(second);
+                if (second == null) return true;
```

Comment:
could you have `return true` on new line? It is easy to misread specially because there is a return on the next line.

Reply:
Ditto. It's just `git revert <commit-id>`